src/js/components/PlacementConstraintsSchemaField.js

```diff
@@ -83,6 +110,20 @@ export default class PlacementConstraintsSchemaField extends Component {
     const { label, errorMessage } = this.props;
     const { batch } = this.state;
 
+    const appendToList = (memo, transaction) => memo.concat(transaction);
```

Comment:
@orlandohohmeier said:

🎨 We might wanna add a some or every operator to the batch so that we can use a simple predicate instead of reducing the batch to an array and then iterating over it. /cc @Poltergeist @nLight

Reply:
I could have reduced over to a boolean instead of a list. I actually had to use reduce here to implement map. Before we think about `some` and `every` (which are I believe more high-level abstractions), we should think of `filter`, but before filter, I think we actually need `map` (naturally we can also always reduce to achieve all of the above).

I also think the API of the Validator should not be `validate(batch)`, `but validate(list)`. Unless we see batch as a list subtype.